src/Page.js

```diff
@@ -138,6 +138,9 @@ function formatSiteNav(renderedSiteNav) {
   // Tidy up the style of the unordered list <ul>
   listItems.parent().attr('style', 'list-style-type: none; margin-left:-1em');
 
+  // highlight current page
+  listItems.find(`a[href='{{baseUrl}}/${src.replace(/\.(md|mbd)$/, '.html')}']`).attr('class', 'current');
```

Comment:
- Begin comments with a capital letter: `// highlight` → `// Highlight`
- Instead of `.attr('class', 'current')`, which replaces the `class`, use `.addClass('current')`.
- Where do we remove the `class`?
- Let's assign `${src.replace(/\.(md|mbd)$/, '.html')}` to a constant since it's clearer and also used below.

Reply:
>> Where do we remove the class?
>
> We don't, the class is needed for `#site-nav a.current` in `site-nav.css` to be applied

Ah right, I was thinking of page nav. For site nav, a page will always be `current`.